Source/Particles/Collision/ScatteringProcess.H

```diff
@@ -42,10 +42,10 @@ public:
 
     ~ScatteringProcess() = default;
 
-    ScatteringProcess (ScatteringProcess const&)            = delete;
-    ScatteringProcess& operator= (ScatteringProcess const&) = delete;
-    ScatteringProcess (ScatteringProcess &&)                = default;
-    ScatteringProcess& operator= (ScatteringProcess &&)     = default;
+    // ScatteringProcess (ScatteringProcess const&)            = delete;
+    // ScatteringProcess& operator= (ScatteringProcess const&) = delete;
+    // ScatteringProcess (ScatteringProcess &&)                = default;
+    // ScatteringProcess& operator= (ScatteringProcess &&)     = default;
```

Comment:
Can we try this first?
```suggestion
    ScatteringProcess (ScatteringProcess &&)                = default;
    ScatteringProcess& operator= (ScatteringProcess &&)     = default;
```